src/repository.ts

```diff
@@ -10,8 +10,20 @@ export const defaultValues: WritableBaseParams = {
 
   // hidden
   //co2emissions: 739, // in 2020, source https://www.bmu.de/pressemitteilung/treibhausgasemissionen-sinken-2020-um-87-prozent/
-  unemployment: 2695, // in 2020, source https://www.arbeitsagentur.de/news/arbeitsmarkt-vorjahre
+  numberOfCitizens: 83157, // Tsd people in 2020, source https://www.destatis.de/DE/Themen/Gesellschaft-Umwelt/Bevoelkerung/Bevoelkerungsstand/Tabellen/zensus-geschlecht-staatsangehoerigkeit-2020.html
+  unemployment: 2695, // Tsd people in 2020, source https://www.arbeitsagentur.de/news/arbeitsmarkt-vorjahre
   gdp: 3332, // in 2020, source http://www.statistikportal.de/de/bruttoinlandsprodukt-vgr
+  
+  publicTransportUsage: 10400,  // Mio rides in 2019, source https://www.vdv.de/daten-fakten.aspx
+  publicTransportPassengerKm: 10400,  // Mio rides in 2019, source https://www.vdv.de/daten-fakten.aspx
```

Comment:
Should the unit be Mio km?

Reply:
You're right, I corrected that and in fact replaced that by the sum of train, bus and metro passenger kms